xbmc/pvr/windows/GUIWindowPVRBase.h

```diff
@@ -112,6 +115,12 @@ namespace PVR
   private:
     bool OpenChannelGroupSelectionDialog(void);
 
+    bool HasChannelGroupsControl();
+    bool IsChannelGroupsControlFocused();
+    void InitChannelGroupsControl();
+    bool SelectActiveChannelGroup();
```

Comment:
same would be nice here `ChannelGroupsControlSelectActiveGroup`

Reply:
No.

Reply:
The introduced method names `SelectActiveChannelGroup` and `ActivateSelectedChannelGroup` aren't self-explained (context is not clear), so why not change it to something that helps understanding the code. My first idea was to put this into a separate class but adding `ChannelGroupsControl` would be enough.

Reply:
I don't think I will change that.